src/librustc/mir/mod.rs

```diff
@@ -818,12 +818,16 @@ pub enum StatementKind<'tcx> {
     /// End the current live range for the storage of the local.
     StorageDead(Lvalue<'tcx>),
 
+    /// Execute a piece of inline Assembly.
     InlineAsm {
         asm: Box<InlineAsm>,
         outputs: Vec<Lvalue<'tcx>>,
         inputs: Vec<Operand<'tcx>>
     },
 
+    /// Assert the given lvalues to be valid inhabitants of their type.
```

Comment:
I think we could extend this comment. For example, it's worth saying that this is part of an experimental attempt at giving semantics to unsafe code, and these statements are not generated during normal MIR execution. Maybe include a link to your blog post. =)